core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -19,11 +19,13 @@ package org.apache.spark
 
 // scalastyle:off
 import java.io.File
+import java.nio.file.Path
 import java.util.{Locale, TimeZone}
 
 import org.apache.log4j.spi.LoggingEvent
```

Comment:
seems the import order is wrong. This should appear after `scala.annotation.tailrec`

Reply:
Not sure why the scala style check has been disabled for imports. I have re-organized them.